apps/remix-ide/src/app/tabs/runTab/settings.js

```diff
@@ -53,6 +53,17 @@ class SettingsUI {
     })
   }
 
+  validateValue () {
+    this.el.querySelector('#value').value = parseInt(this.el.querySelector('#value').value)
+    // assign 0 if given value is
+    // - empty
+    // - not valid (for ex 4345-54)
+    // - contains only '0's (for ex 00..0)
+    if (!this.el.querySelector('#value').value) this.el.querySelector('#value').value = 0
+    // if giveen value is negative, ignore '-'
+    if (this.el.querySelector('#value').value < 0) this.el.querySelector('#value').value = Math.abs(this.el.querySelector('#value').value)
```

Comment:
You should probably convert `this.el.querySelector('#value').value` to number before making comparison to zero (0), just to be safe.

Reply:
it is a number. the types of input is number